go/libkb/upgrade_instructions.go

```diff
@@ -58,6 +58,16 @@ func darwinUpgradeInstructions() {
 	// TODO: non-brew update instructions
 }
 
+func windowsUpgradeInstructions() {
+	serverURI := ProductionServerURI
+	if DefaultRunMode == DevelRunMode {
+		serverURI = DevelServerURI
+	} else if DefaultRunMode == StagingRunMode {
+		serverURI = StagingServerURI
+	}
+
+	G.Log.Warning("To upgrade, download the latest Keybase installer from " + serverURI)
```

Comment:
I guess we should plumb it through from the server via the API.

On Sunday, November 29, 2015, Steve Sanders <notifications@github.com>
wrote:

> In go/libkb/upgrade_instructions.go
> <https://github.com/keybase/client/pull/1410#discussion_r46096096>:
>
> > @@ -58,6 +58,16 @@ func darwinUpgradeInstructions() {
> >  	// TODO: non-brew update instructions
> >  }
> >
> > +func windowsUpgradeInstructions() {
> > +	serverURI := ProductionServerURI
> > +	if DefaultRunMode == DevelRunMode {
> > +		serverURI = DevelServerURI
> > +	} else if DefaultRunMode == StagingRunMode {
> > +		serverURI = StagingServerURI
> > +	}
> > +
> > +	G.Log.Warning("To upgrade, download the latest Keybase installer from " + serverURI)
>
> That is just keybase.io. If we want to hardcode in an URL, what should it
> be? I just figured it was easier and more flexible not to, and whoever is
> smart enough to download it once could figure out where to update from.
> … <#15154875f0242192_>
> On Sat, Nov 28, 2015 at 2:30 PM, Maxwell Krohn <notifications@github.com
> <javascript:_e(%7B%7D,'cvml','notifications@github.com');>> wrote: In
> go/libkb/upgrade_instructions.go <#1410 (comment)
> <https://github.com/keybase/client/pull/1410#discussion_r46085280>>: > @@
> -58,6 +58,16 @@ func darwinUpgradeInstructions() { > // TODO: non-brew
> update instructions > } > > +func windowsUpgradeInstructions() { > +
> serverURI := ProductionServerURI > + if DefaultRunMode == DevelRunMode { >
> + serverURI = DevelServerURI > + } else if DefaultRunMode == StagingRunMode
> { > + serverURI = StagingServerURI > + } > + > + G.Log.Warning("To upgrade,
> download the latest Keybase installer from " + serverURI) is this a direct
> link to the page, or just the home page? Can you provide direct link to the
> install page? Also, we don't need to get this right for staging or devel. —
> Reply to this email directly or view it on GitHub <
> https://github.com/keybase/client/pull/1410/files#r46085280>.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/keybase/client/pull/1410/files#r46096096>.
>